Source/WebCore/platform/graphics/WidthIterator.cpp

```diff
@@ -541,10 +541,7 @@ bool WidthIterator::characterCanUseSimplifiedTextMeasuring(UChar character, bool
         break;
     }
 
-    if (character >= HiraganaLetterSmallA
-        || u_charType(character) == U_CONTROL_CHAR
-        || (character >= nullCharacter && character < space)
-        || (character >= deleteCharacter && character < noBreakSpace))
+    if (character >= HiraganaLetterSmallA || isControlCharacter(character))
```

Comment:
I'm assuming you checked to make sure these are equivalent

Reply:
I did. See [List of Unicode Characters of Category “Control”](https://www.compart.com/en/unicode/category/Cc).